open-company/open-company-web
# Section settings button

Card: https://trello.com/c/UQIRNl4M

Change section settings button to make it more visible.

Abstract (see comment tagged for bago2k4) https://share.goabstract.com/9c0bfdca-2fba-42b8-9293-cd3509937304

Updated design for private and public sections: https://share.goabstract.com/c3d70041-db90-432c-a609-b3386bc03603

To test:
- on desktop
- check AP
- [x] do you NOT see the section settings gear? Good
- go to a random section
- [x] do you see the section settings gear? Good
- click it
- on mobile
- go to AP
- [x] do you NOT see the section settings gear? Good
- go to a random section
- [x] do you NOT see the section settings gear? Good
